XRT DEM Iterative Solver Module (#367)
HinodeXRT/xrtpy: docs/changelog/0.6.0.rst

XRTpy v0.6.0
=========================

New Features
------------
- Introduced :class:`~xrtpy.xrt_dem_iterative.XRTDEMIterative`, a Python

>   implementation of the IDL routine ``xrt_dem_iterative2.pro``. This solver

Reviewer: ```suggestion
  implementation of the IDL routine :file:`xrt_dem_iterative2.pro`. This solver
```

  uses spline-parameterized DEM curves and iterative least-squares fitting to

>   infer the differential emission measure from Hinode/XRT multi-filter

Reviewer: ```suggestion
  infer the DEM from Hinode/XRT multi-filter
```

  observations. (`#PR <https://github.com/HinodeXRT/xrtpy/pull/PR>`__)
- Added Monte Carlo uncertainty estimation to the DEM solver through the
  ``monte_carlo_runs`` parameter. Perturbed intensity realizations are used

>   to estimate the spread in DEM(T).

Reviewer: ```suggestion
  to estimate the spread in :math:`\mathrm{DEM}(T)`.
```
